include/podio/CollectionBuffers.h

```diff
@@ -25,6 +25,7 @@ using VectorMembersInfo = std::vector<std::pair<std::string, void*>>;
  */
 struct CollectionWriteBuffers {
   void* data{nullptr};
+  void* vecPtr{nullptr};
```

Comment:
Not entirely happy with this one. But at the moment we seem to not have any other option(?).

Reply:
Not yet, it's in my todo list